schema/denm/denm_schema_2-1-0.json

```diff
@@ -0,0 +1,579 @@
+      "name": "DENM TS 103 831",
+      "version": "2.2.1",
+      "url": "https://forge.etsi.org/rep/ITS/asn1/denm_ts103831/-/tree/v2.2.1"
+    },
+    {
+      "name": "DENM EN 302 637-3",
+      "version": "1.3.1",
+      "url": "https://forge.etsi.org/rep/ITS/asn1/denm_en302637_3/-/tree/v1.3.1"
+    }
+  ],
+  "type": "object",
+  "additionalProperties": false,
+  "required": [
+    "message_type",
+    "source_uuid",
+    "timestamp",
+    "version",
+    "message"
+  ],
+  "properties": {
+    "message_type": {
+      "type": "string",
+      "description": "Type of the message carried in 'message' property",
+      "const": "denm"
+    },
+    "source_uuid": {
+      "type": "string",
+      "examples": [
+        "UNKNOWN",
+        "CCU6",
+        "MEC2"
+      ]
+    },
+    "timestamp": {
+      "type": "integer",
+      "description": "Unit: millisecond. The timestamp when the message was generated since Unix Epoch (1970/01/01)",
+      "examples": [
+        1574778515424
+      ],
+      "minimum": 1514764800000,
+      "maximum": 1830297600000
+    },
+    "version": {
+      "type": "string",
+      "description": "JSON message format version",
+      "const": "2.0.0"
+    },
+    "path": {
+      "type": "array",
+      "description": "List of ordered elements root source of the message",
+      "minItems": 1,
+      "items": {
+        "type": "object",
+        "required": [
+          "position",
+          "message_type"
+        ],
+        "properties": {
+          "position": {
+            "type": "object",
+            "required": [
+              "latitude",
+              "longitude",
+              "altitude"
+            ],
+            "properties": {
+              "latitude": {
+                "type": "integer",
+                "description": "Unit: 0.1 microdegree. oneMicrodegreeNorth (10), oneMicrodegreeSouth (-10), unavailable(900000001)",
+                "default": 900000001,
+                "minimum": -900000000,
+                "maximum": 900000001
+              },
+              "longitude": {
+                "type": "integer",
+                "description": "Unit: 0.1 microdegree. oneMicrodegreeEast (10), oneMicrodegreeWest (-10), unavailable(1800000001)",
+                "default": 1800000001,
+                "minimum": -1800000000,
+                "maximum": 1800000001
+              },
+              "altitude": {
+                "type": "integer",
+                "description": "Unit: 0.01 meter. referenceEllipsoidSurface(0), oneCentimeter(1), unavailable(800001)",
+                "default": 800001,
+                "minimum": -100000,
+                "maximum": 800001
+              }
+            }
+          },
+          "message_type": {
+            "description": "root message type source of the element",
+            "enum": [
```

Comment:
Change to 
`"type": "string"`
`"const": "denm"`

Reply:
Are you sure ?
As I understanc if, these objects are part of the list of the originating messages that lead to creating the DENM
So for example, a list of static CAM one behind the other identified as a TrafficCondition

Reply:
My bad, I was mistaken with the root level `message_type`... OK in this case then.